src/patternfly/components/Menu/menu.scss

```diff
@@ -187,6 +199,25 @@
   }
 }
 
+// breadcrumb
+.pf-c-menu__breadcrumb {
+  display: flex;
+  align-items: center;
+  min-height: var(--pf-c-menu__breadcrumb--MinHeight);
```

Comment:
could you use negative margins (-1 * --spacer--form-control) on the `pf-c-breadcrumb__dropdown` element instead?

Reply:
then if the dropdown appears in some other context, the breadcrumb height won't change